core/board.py

```diff
@@ -27,11 +25,21 @@ def update(self, user_shot_choice, is_hit):
                 'I': 8,
                 'J': 9,
                 }
+
+    def update(self, user_shot_choice, is_hit):
         user_letter = user_shot_choice[0]
         user_num = user_shot_choice[1:]
         
-        self.state[rows[user_num]][columns[user_letter]] = 'Hit' if is_hit else 'Miss' 
-            
+        if is_hit == 'Hit':
```

Comment:
it seems like you could completely bypass this conditional but just doing something like `self.state[self.rows[user_num]][self.columns[user_letter]] = is_hit` but then I'd expect that functions name to change as, to me, it currently indicates a boolean response